samples/nrf9160/modem_shell/README.rst

```diff
@@ -541,6 +541,8 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
+See :ref:`cmake_options` for instructions on how to provide CMake options, for example to use a configuration overlay.
```

Comment:
I'm afraid the linked piece of doc (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_modifying.html#cmake-options) does not give instructions on using overlay files.

Reply:
That link gives instructions how to add CMake options (`-D` in our case) after the west build command, which is the `--` syntax after which you can add `-D` parameters and overlay (`OVERLAY_CONFIG`) is just a compiler definition.

Reply:
This is a general sentence copied from somewhere and it can be improved if you know how but let's at least not add too much instructions into this document on such a general thing.

Reply:
Fair enough. Let's keep it as it is.